src/components/structure/interacting/secondary-navigation/secondary-navigation.tsx

```diff
@@ -25,6 +25,11 @@ const SecondaryNavigation = () => {
     collection.showCollection();
   }
 
+  const hideSidebar = () => {
```

Comment:
Es gibt die Möglichkeit die Sidebar auszublenden, nur kann man sie nach dem Ausblenden nicht mehr wieder einblenden.
Oder habe ich etwas in die Richtung übersehen?